conf-testenv/config.yaml

```diff
@@ -21,7 +21,7 @@ cert_file: "conf/idp.crt"
 # cfr. https://pysaml2.readthedocs.io/en/latest/howto/config.html#metadata
 metadata:
   remote:
-    - "http://hub-spid-login-ms:9090/metadata"
+    - "http://hub-spid-login-ms:8080/metadata"
```

Comment:
AFAIK port 8080 is already taken by `spid-saml-check`. is it right?

Reply:
Yes, it is. But the port is used inside a `docker-compose` network, which works like a DNS, so the origins `http://hub-spid-login:8080` and `http://spid-saml-check:8080` _inside_ the network don't collide. Obviously if you map both externally with the `ports` directive they get both mapped to `localhost:8080` and _externally_ collide. It's a matter of chioce. Anyway, you can always map `8080` externally to another port so as to avoid also external collision

Reply:
Ok I've mapped said-saml-check to `9191` port in order to avoid confusion :D